packages/components/src/context-actions/ContextActionUtils.ts

```diff
@@ -116,20 +116,9 @@ class ContextActionUtils {
    * @returns Promise Resolved on success, rejected on failure
    */
   static async copyToClipboard(text: string): Promise<void> {
-    try {
-      const permissions = await navigator.permissions.query({
-        name: 'clipboard-write' as PermissionName,
-      });
-      if (permissions.state !== 'granted' && permissions.state !== 'prompt') {
-        throw new Error('Invalid permissions for clipboard-write');
-      }
-    } catch (e) {
-      // Fallback if we can't get permissions for some reason
+    await navigator.clipboard.writeText(text).catch(err => {
```

Comment:
You shouldn't need the `await` here any more or the `async` in the function name. Just `return navigator.clipboard...`